### ISSUE TITLE
Initializing when stopping docker compose and starting it again

### ISSUE THREAD

**Issue by titansmc** [CachetHQ/Docker#302]
Hi,
I am not sure under which circumstances (maybe always) when I do docker-compose down, and then docker-compose up I get the setup page and  I lose the data. Is there something I am missing ? 

P.S. I am mounting always the DDBB directory from the same place, locally, not using volume.

**Comment by shuichiro-makigaki** [comment]
There are three ways to avoid the issue.
https://stackoverflow.com/a/39175305 is helpful.

**Comment by Disinterpreter** [comment]
No, it's because there's no ways in documentation to describe how to mount the service container =\

I don't understand what I need mount to it works without erasing.

**Comment by djdefi** [comment]
Are you able to provide the full output from `docker-compose up`? I suspect the log entries would help to determine what isn't being picked up.

**Comment by Disinterpreter** [comment]
> 
> 
> Are you able to provide the full output from `docker-compose up`? I suspect the log entries would help to determine what isn't being picked up.

I solved it, because I know how it works. Please, make a directory for db and change the line https://github.com/CachetHQ/Docker/blob/master/docker-compose.yml#L7

Or write about it in documentation
